lib/utils/utils.ts

```diff
@@ -65,4 +61,9 @@ utils.groupBy = (arr: object[], keyGetter: (item: object) => string | number): o
   return ret;
 };
 
-export default utils;
+export {
+  getTsString,
```

Comment:
if you don't want to export them united as the default export, then it's better to do `export const ...` on each one of them.

Reply:
Done.